src/dataframe/dataframe.jl

```diff
@@ -1024,15 +930,23 @@ select!(df::DataFrame, c::Any) = select!(df, index(df)[c])
 
 """
     select(df::DataFrame, inds, copycols::Bool=true)
+    select(dfv::SubDataFrame, inds, copycols::Bool=true)
```

Comment:
Better have a single signature for `AbstractDataFrame`? Then the text can mention that `copycols` isn't supported for `DataFrameRow` (in theory we could allocate a new parent though if we wanted).

Reply:
OK. For `DataFrameRow` I thought it is completely not useful and the default should be `copycols=false` (as opposed to the other methods) anyway so I thought that it is better not to support it.

Reply:
Maybe don't support `select` on `DataFrameRow` at all? It sounds problematic to use a different default, and anyway a `DataFrameRow` isn't a data frame.

Reply:
OK